deploy/role.yaml

```diff
@@ -22,6 +23,12 @@ rules:
   - create
   - update
   - delete
+- apiGroups:
```

Comment:
In this case we need to remove hack with Role for Travis CI. Now utilizing it for the SA

Reply:
Doesn't that also require the "list" and "watch" verbs? That's why I didn't update before.

Reply:
Need to be validated, we can remove it and look at the debug-pods.sh processing in the Travis CI